taskcluster/fenix_taskgraph/transforms/mark_as_shipped.py

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the build
+kind.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def make_task_description(config, jobs):
+    for job in jobs:
+        product = "Fenix"
+        version = config.params['release_version'] or "{ver}"
```

Comment:
Question: What is `"{ver}"` in this context? I don't see it defined in gecko https://searchfox.org/mozilla-central/rev/b243debf6235b050b42fd2eb615fdc729636ca6b/taskcluster/taskgraph/transforms/release_mark_as_shipped.py

Reply:
I used it as a clear "this isn't right" designation, because default `release_version` is `""` so it was generating a `Fenix--build1` thing by default. -- I can remove it if this is less clear.

Reply:
Ah, I see! I agree the magic string method is a good way to go. Maybe we should set it to something that shows `release_version` was missing and it's obviously wrong. How about something like `"MISSING_VERSION"`?